subnet/config_test.go

```diff
@@ -35,8 +35,8 @@ func TestConfigDefaults(t *testing.T) {
 		t.Errorf("SubnetMin mismatch, expected 10.3.1.0, got %s", cfg.SubnetMin)
 	}
 
-	if cfg.SubnetMax.String() != "10.3.255.0" {
-		t.Errorf("SubnetMax mismatch, expected 10.3.255.0, got %s", cfg.SubnetMax)
+	if cfg.SubnetMax.String() != "10.3.2.0" {
```

Comment:
This does not seem right. The expected subMax should be a 10.3.255.0. So subMin-subMax(10.3.1.0->10.3.255.0) is about a whole subnet (/16).